### PR TITLE
Updates copyright statements and some translation (GUI)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

1. License file and footer: In the copyright statement a range of years is given. That requires updates, but year is not needed.
2. Licens file: Nothing is said about copyright for contributors.
3. The copyright statement in the footer is not translated.
4. Untranslated or incorrectly translated strings found in the same file as the copyright statement goes.

## Changes

1. Removes year from copyright and give the correct "translation" of Internetstiftelsen (LICENSE).
2. Adds statement about contributor's copyright (LICENSE).
3. Adds translation mechanism of the statement in the footer (footer.component.html)
4. Adds translation data (i18n/*.json)
5. Updated translation in some of i18n/*.json file where translation was missing or incorrect  

## How to test this PR

1. Check that the footer is correctly translated in all languages.
2. Check that Danish and Swedish are correctly translated where changed.

### CODE REVIEW

**Comment by matsduf** [comment]
This PR deprecates #209.

**Comment by samisalmensuo** [comment]
> This is about the copyright statement in the footer of the GUI. In English it says:
> 
> > Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved
> 
> In Swedish I have translated that to:
> 
> > © AFNIC och Internetstiftelsen. Alla rättigheter förbehållna"
> 
> @haarbo, what should it be in Danish?
> 
> @samisalmensuo, what should be in Finnish?
> 
> @vlevigneron and @pnax, what should it be in French?
> 
> @repperille, what should it be in Norwegian?
> 
> Please provide your translation in a comment. We use "The Swedish Internet Foundation" for all non-Swedish contexts.

Finnish:
© AFNIC ja IIS. Kaikki oikeudet pidätetään

**Comment by matsduf** [comment]
> © AFNIC ja IIS. Kaikki oikeudet pidätetään

Thank you. I have updated with your translation (but using "The Swedish Internet Foundation" instead of "IIS").

**Comment by repperille** [comment]
Norwegian:
© AFNIC og The Swedish Internet Foundation.

**Comment by haarbo** [comment]
Hello!


© AFNIC og den Svenske Internetstiftelse. Alle rettigheder forbeholdes.



Best regards

Niels Haarbo
Seniorkonsulent / Senior Consultant
This is an email from DIFO/DK Hostmaster A/S. This message may contain confidential information and is intended solely for the use of the intended addressee. If you are not the intended addressee, please notify the sender immediately and delete this e-mail from your system.

**Comment by pnax** [comment]
> what should it be in French?

I tried to directly push the change to your branch, and this seems to work. I hope you don't mind :)

**Comment by matsduf** [comment]
> I tried to directly push the change to your branch, and this seems to work. I hope you don't mind :)

That is fine.

I saw that you replaced back to HT. I have a question. Should we really have HT (horizontal tab, U+0009) in the file instead of SP (space, U+0020) in that file? A problem with HT is that it is displayed differently in different environment. It is not standardized how much it should "push in". With SP and monospace font it will be the same in all environments (as far as I know).

**Comment by pnax** [comment]
> I saw that you replaced back to HT. [...]

I agree, I too like spaces better than tabulations. However the fr.json file is using tabs, so I choose to make it consistent for this new line. This is totally fine for me to change to spaces (but then I think all the lines should use spaces).

(and it looks like this is the same for all the *.json file, they all use tabs instead of spaces, beside this new copyright line)

**Comment by matsduf** [comment]
> © AFNIC og den Svenske Internetstiftelse. Alle rettigheder forbeholdes.

@haarbo, thank you. I have updated the translation (but replaced "den Svenske Internetstiftelse" with "The Swedish Internet Foundation" because that is how it wants it to be).

Now I found a few more strings in the same file that are not translated into Danish. Could you provide translation for them too? Just as a message here is fine.

* "Group by modules"
* "Level"
* "Link"
* "Message"
* "Notice! More info on undelegated test"
* "Options"
* "Please set the api endpoint"
* "Reset"
* "Run test"
* "There is no delegation for the zone"

**Comment by matsduf** [comment]
> I agree, I too like spaces better than tabulations. However the fr.json file is using tabs, so I choose to make it consistent for this new line. This is totally fine for me to change to spaces (but then I think all the lines should use spaces).
> 
> (and it looks like this is the same for all the *.json file, they all use tabs instead of spaces, beside this new copyright line)

I agree on consistency, but I did no change from HT to SP consciously. I will fix so that it is consistent.

**Comment by haarbo** [comment]
*   "Group by modules"
Gruppér efter moduler

  *   "Level"
Niveau

  *   "Link"
Link

  *   "Message"
Besked

  *   "Notice! More info on undelegated test"
Bemærk! Mere info om udelegeret test

  *   "Options"
Muligheder

  *   "Please set the api endpoint"
Vælg venligst api endpoint.

  *   "Reset"
Nulstil

  *   "Run test"
”Afvikl test” eller ”Kør test”

  *   "There is no delegation for the zone"
Der er ikke nogen delegation for zonen.

**Comment by matsduf** [comment]
> Norwegian:
> © AFNIC og The Swedish Internet Foundation.

Thanks. I have updated with your translation.

**Review by pamasse — APPROVED** [review]

**Comment by matsduf** [comment]
@mattias-p and @pnax, Travis complaints, but that is unrelated. I have verified that the update works as expected. Can I merge anyway? Can you approve?

File: LICENSE [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ### Code license
 
-Copyright (c) 2013-2020, The Swedish Internet Foundation
-Copyright (c) 2013-2020, AFNIC
+Copyright (c) The Swedish Internet Foundation (<https://internetstiftelsen.se/en/>)
+Copyright (c) AFNIC (<https://www.afnic.fr/en/>)
```
Comment (mattias-p) [inline LICENSE:4]: Why are the links marked up with both parentheses and angle brackets? Isn't it enough with just one of them?

**Review by mattias-p — APPROVED** [review]

**Review by pnax — APPROVED** [review]
Good for me to merge event though Travis complains.

File: LICENSE (hunk 1 above)
Comment (matsduf) [reply LICENSE:4]: In this case I guess it is, but that is consistent with the LICENSE file in the other repositories.

**Pull request merged**